.goreleaser.yaml

```diff
@@ -9,14 +9,18 @@ before:
     - go mod tidy
     # These hooks get run once. We want to explicitly download every embedded binary so we can build the multiplexer
     # for every architecture. These versions must be updated at the same time as the version in the Makefile.
-    - ./scripts/download_binary.sh celestia-app_Darwin_arm64.tar.gz celestia-app_darwin_v3_arm64.tar.gz v3.10.5
-    - ./scripts/download_binary.sh celestia-app_Linux_arm64.tar.gz celestia-app_linux_v3_arm64.tar.gz v3.10.5
-    - ./scripts/download_binary.sh celestia-app_Darwin_x86_64.tar.gz celestia-app_darwin_v3_amd64.tar.gz v3.10.5
-    - ./scripts/download_binary.sh celestia-app_Linux_x86_64.tar.gz celestia-app_linux_v3_amd64.tar.gz v3.10.5
-    - ./scripts/download_binary.sh celestia-app-standalone_Darwin_arm64.tar.gz celestia-app_darwin_v4_arm64.tar.gz v4.1.0-arabica
-    - ./scripts/download_binary.sh celestia-app-standalone_Linux_arm64.tar.gz celestia-app_linux_v4_arm64.tar.gz v4.1.0-arabica
-    - ./scripts/download_binary.sh celestia-app-standalone_Darwin_x86_64.tar.gz celestia-app_darwin_v4_amd64.tar.gz v4.1.0-arabica
-    - ./scripts/download_binary.sh celestia-app-standalone_Linux_x86_64.tar.gz celestia-app_linux_v4_amd64.tar.gz v4.1.0-arabica
+    - ./scripts/download_v3_binary.sh celestia-app_Darwin_arm64.tar.gz celestia-app_darwin_v3_arm64.tar.gz v3.10.5
+    - ./scripts/download_v3_binary.sh celestia-app_Linux_arm64.tar.gz celestia-app_linux_v3_arm64.tar.gz v3.10.5
+    - ./scripts/download_v3_binary.sh celestia-app_Darwin_x86_64.tar.gz celestia-app_darwin_v3_amd64.tar.gz v3.10.5
+    - ./scripts/download_v3_binary.sh celestia-app_Linux_x86_64.tar.gz celestia-app_linux_v3_amd64.tar.gz v3.10.5
+    - ./scripts/download_v4_binary.sh celestia-app-standalone_Darwin_arm64.tar.gz celestia-app_darwin_v4_arm64.tar.gz v4.1.0-arabica
+    - ./scripts/download_v4_binary.sh celestia-app-standalone_Linux_arm64.tar.gz celestia-app_linux_v4_arm64.tar.gz v4.1.0-arabica
+    - ./scripts/download_v4_binary.sh celestia-app-standalone_Darwin_x86_64.tar.gz celestia-app_darwin_v4_amd64.tar.gz v4.1.0-arabica
+    - ./scripts/download_v4_binary.sh celestia-app-standalone_Linux_x86_64.tar.gz celestia-app_linux_v4_amd64.tar.gz v4.1.0-arabica
+    - ./scripts/download_v5_binary.sh celestia-app-standalone_Darwin_arm64.tar.gz celestia-app_darwin_v5_arm64.tar.gz v5.0.1-arabica
+    - ./scripts/download_v5_binary.sh celestia-app-standalone_Linux_arm64.tar.gz celestia-app_linux_v5_arm64.tar.gz v5.0.1-arabica
+    - ./scripts/download_v5_binary.sh celestia-app-standalone_Darwin_x86_64.tar.gz celestia-app_darwin_v5_amd64.tar.gz v5.0.1-arabica
+    - ./scripts/download_v5_binary.sh celestia-app-standalone_Linux_x86_64.tar.gz celestia-app_linux_v5_amd64.tar.gz v5.0.1-arabica
```

Comment:
[could be a flup or handled here, just needs to be handled before cutting a release]

I don't think this scripts exist, we only have the download binary script

maybe we could use the make command remove one copy

Reply:
as soon as we get a second to breath we should really prioritize #4921